x-pack/test/observability_ai_assistant_functional/tests/feature_controls/settings_security.spec.ts

```diff
@@ -60,22 +61,63 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       it('allows updating of an advanced setting', async () => {
-        const testSearchConnectorIndexPattern = 'my-logs-index-pattern';
+        await PageObjects.common.navigateToUrl('obsAIAssistantManagement', '', {
+          ensureCurrentUrl: false,
+          shouldLoginIfPrompted: false,
+          shouldUseHashForSubUrl: false,
+        });
+        const testSearchConnectorIndexPattern = 'my-search-index-pattern';
         const searchConnectorIndexPatternInput = await testSubjects.find(
           ui.pages.settings.searchConnectorIndexPatternInput
         );
+        // make sure the input is empty (default value)
         await searchConnectorIndexPatternInput.clearValue();
         await searchConnectorIndexPatternInput.type(testSearchConnectorIndexPattern);
         const saveButton = await testSubjects.find(ui.pages.settings.saveButton);
         await saveButton.click();
-        await browser.refresh();
-        const searchConnectorIndexPatternInputValue =
-          await searchConnectorIndexPatternInput.getAttribute('value');
-        expect(searchConnectorIndexPatternInputValue).to.be(testSearchConnectorIndexPattern);
-        // reset the value
-        await searchConnectorIndexPatternInput.clearValue();
-        await searchConnectorIndexPatternInput.type('logs-*');
+        // wait for page to refrsh
+        await testSubjects.missingOrFail(ui.pages.settings.searchConnectorIndexPatternInput, {
+          timeout: 2000,
+        });
+        // wait for the new page to fully load
+        await testSubjects.existOrFail(ui.pages.settings.searchConnectorIndexPatternInput, {
+          timeout: 2000,
+        });
+        expect(await searchConnectorIndexPatternInput.getAttribute('value')).to.be(
+          testSearchConnectorIndexPattern
+        );
+        // reset the value back to default
+        const resetToDefaultLink = await testSubjects.find(ui.pages.settings.resetToDefaultLink);
+        await resetToDefaultLink.click();
+
+        expect(await searchConnectorIndexPatternInput.getAttribute('value')).to.be('');
         await saveButton.click();
+        await testSubjects.missingOrFail(ui.pages.settings.searchConnectorIndexPatternInput, {
+          timeout: 2000,
+        });
+        // wait for the new page to fully load
+        await testSubjects.existOrFail(ui.pages.settings.searchConnectorIndexPatternInput, {
+          timeout: 2000,
+        });
+        expect(await searchConnectorIndexPatternInput.getAttribute('value')).to.be('');
+      });
+      it('displays failure toast on failed request', async () => {
+        const searchConnectorIndexPatternInput = await testSubjects.find(
+          ui.pages.settings.searchConnectorIndexPatternInput
+        );
+        await searchConnectorIndexPatternInput.clearValue();
+        await searchConnectorIndexPatternInput.type('test');
+
+        await interceptRequest({ driver: driver.driver, pattern: '*kibana\\/settings*' }, (responseFactory) => {
```

Comment:
You'll also need to update `interceptRequest`

Reply:
Thanks, I've reverted the change for now. I'm going to open up a separate PR for this.  I'm going to remove our version and use the one added to the `browser` service, which date_usage plugin also uses.  I'll update the other parts of our code to use it as well.